client/src/components/calendar/CalendarDate.tsx

```diff
@@ -0,0 +1,82 @@
+import { SELECT_STATUS, SelectStatus } from './calendar.core';
+
+type Props = {
+  size?: 'small' | 'large';
+  date: Date;
+  selectStatus: SelectStatus;
+  onClick?: () => void;
+};
+
+function CalendarDate({ size = 'small', date, selectStatus, onClick }: Props) {
+  switch (selectStatus) {
+    case SELECT_STATUS.NONE:
+      return <div className="flex aspect-square w-full items-center justify-center" />;
+
+    case SELECT_STATUS.UNSELECTABLE:
+      return <div className="flex aspect-square w-full items-center justify-center">{date.getDate()}</div>;
```

Comment:
선택할 수 없는 날짜의 경우 text 색상을 회색으로 변경하여 선택 가능한 날짜와 불가능한 날짜를 한 눈에 구분할 수 있도록 하면 어떨까요?

```suggestion
    case SELECT_STATUS.UNSELECTABLE:
      return <div className="flex aspect-square w-full items-center justify-center text-background-300">{date.getDate()}</div>;
```
<img width="1796" alt="무제" src="https://github.com/softeerbootcamp-3rd/Team9-HexaCore/assets/90602694/ee08f836-9c48-413c-ab56-36aca3902dac">